tests/end2end/features/invoke.feature

```diff
@@ -0,0 +1,111 @@
+Feature: Invoking a new process
+    Simulate what happens when running qutebrowser with an existing instance
+
+    Background:
+        Given I clean up open tabs
+        And I set general -> new-instance-open-target to tab
```

Comment:
No, good point. I originally just added it to begin with to make sure I always had the right default set, but it's better to have all the tests do it explicitly either way.